components/MailchimpForm.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/no-danger, no-dupe-keys */
+
+import React, { Component } from 'react';
+
+class MailchimpForm extends Component {
+  render() {
+    return (
+      <form
+        action="https://texasjusticeinitiative.us18.list-manage.com/subscribe/post?u=fd262cb4a5fc0bafb38da2e22&amp;id=2663621fac"
+        method="post"
+        target="_blank"
+      >
+        <input
```

Comment:
These look great on the sidebar, but they get really long on the homepage on desktop. What do you think about setting a max-width for them?

Reply:
Good call! Addressed in https://github.com/texas-justice-initiative/website-nextjs/pull/129/commits/941a68433370ea3c7df0e3759db68ad1fc2e4a6f.